ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml

```diff
@@ -184,7 +184,7 @@ tests:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-aws-ipi-proxy-cco-manual-security-token-service
 - as: e2e-aws-usgov-ipi-private
-  cron: 0 22 */3 * *
```

Comment:
I'm ok to keep it as current. 

Just wanted to change the frequency for `cco-manual` related tests because their selected test cases are the same. Sorry if this isn't clear in the previous PR.

Reply:
Updated